forum/profiles/serializers.py

```diff
@@ -83,3 +83,35 @@ class Meta:
             'city',
             'description',
         ]
+
+
+class ViewedStratupReadSerializer(serializers.ModelSerializer):
```

Comment:
The class name and references use a typo (`ViewedStratup` instead of `ViewedStartup`). This can create confusion and should be corrected to maintain consistency.